include/bout/hypre_interface.hxx

```diff
@@ -130,8 +130,8 @@ public:
 
     location = f.getLocation();
     initialised = true;
-    HypreMalloc(&I, vsize*sizeof(HYPRE_BigInt)); 
-    HypreMalloc(&V, vsize*sizeof(HYPRE_Complex));
+    HypreMalloc(I, vsize*sizeof(HYPRE_BigInt));
+    HypreMalloc(V, vsize*sizeof(HYPRE_Complex));
```

Comment:
@johnomotani In cudaMallocManaged mode of the define the call is expecting a void** (i.e. Handle), so some finesse still needed:  

__host__ ​cudaError_t cudaMallocManaged ( void** devPtr, size_t size, unsigned int  flags = cudaMemAttachGlobal )

Reply:
Ah, OK. Thanks @jonesholger. Trouble with me not being able to test the GPU version...  Without these changes the CPU version refused to compile. Please feel free to push fixes to this branch!